APACHE_PROPOSAL.md

```diff
@@ -6,7 +6,7 @@ HugeGraph is a graph database with high performance and scalability.
 
 ## Proposal
 
-HugeGraph is to provide an large-scale graph database, in the scenario of 100+ billion data which achieves availability and balance between performance and cost, and has complete HTAP capabilities in an internal system.
+HugeGraph is to provide a large-scale graph database, in the scenario of 100+ billion data which achieves availability and balance between performance and cost, and has complete HTAP capabilities in the internal system.
```

Comment:
100+ billion data -> 100+ billion data (vertexes and edges)

Reply:
done